client.js

```diff
@@ -62,30 +65,33 @@ async function sqlExample(hzClient) {
                ('England', 'London'),
                ('Turkey', 'Ankara'),
                ('United States', 'Washington, DC');`;
-    await hzClient.getSql().execute(insertQuery);
+    await client.getSql().execute(insertQuery);
     console.log("The data has been inserted successfully.");
     console.log("--------------------");
+}
 
+async function selectAllCapitals(client){
     console.log("Retrieving all the data via SQL...");
-    const sqlResultAll = await hzClient.getSql()
+    const sqlResultAll = await client.getSql()
         .execute("SELECT * FROM cities", [], {returnRawResult: true});
     for await (const row of sqlResultAll) {
         const country = row.getObject(0);
         const city = row.getObject(1);
         console.log(`${country} - ${city}`);
     }
     console.log("--------------------");
+}
 
+async function selectCapitalNames(client){
     console.log("Retrieving a city name via SQL...");
-    const sqlResultRecord = await hzClient.getSql()
+    const sqlResultRecord = await client.getSql()
         .execute("SELECT __key AS country, this AS city FROM cities WHERE __key = ?", ["United States"]);
     for await (const row of sqlResultRecord) {
         const country = row.country;
         const city = row.city;
         console.log(`Country name: ${country}; City name: ${city}`);
     }
     console.log("--------------------");
-    process.exit(0);
 }
 
 async function jsonSerializationExample(hzClient) {
```

Comment:
Can you pass sqlService to the methods inside this method? Also can you add the comment above this method?

Reply:
Can you update links in comments in `createMappingForCountries` and other methods in jsonSerializationExample ? The links refer to 5.0, they should be 5.1

Reply:
Can you rename populateCountriesMap to populateCountriesWithMap?

Reply:
Can you rename populateCityMap to populateCities?

Reply:
Populations in populateCityMa function is not the same in Java